msi/release.sh

```diff
@@ -87,6 +90,6 @@ wix/light.exe \
 	obj/GitProduct.wixobj \
 	obj/GitComponents.wixobj \
 	-o $TARGET -ext WixUtilExtension \
-	-b / -b ../installer -sval &&
+	-b / -b ../installer &&
```

Comment:
> I did.

Educate me.

Is

> Create a 'Git' folder in the program menu and install the cmd, bash and GUI shortcuts there.

telling me why `-sval` was removed, and what it does, or is it

> Also enables static analysis to catch errors handling per-user folders.

If you thrive on making other people feel like they are stupid, dumb idiots, I congratulate you for making me another target :smile: